### ISSUE TITLE
1.10.2 MO 0.5.1  Crash games during startup

### ISSUE THREAD

**Issue by KiraLinux** [simeonradivoev/MatterOverdrive#631]
[logs.zip](https://github.com/simeonradivoev/MatterOverdrive/files/836188/logs.zip)
Report and logs in the archive

**Comment by AceologyGaming** [comment]
[crash-2017-03-12_12.53.30-client.txt](https://github.com/simeonradivoev/MatterOverdrive/files/836517/crash-2017-03-12_12.53.30-client.txt)
Not just a one off, I'm getting this too

**Comment by KiraLinux** [comment]
Where version 0.5.2?

**Comment by shadowfacts** [comment]
It hasn't been released yet.

**Comment by Muirrum** [comment]
Please for the love of god, use pastebin!

**Comment by joshtayloruk** [comment]
Hey I'm glad I'm not the only one with this issue and it's been noted down as a bug. Hope it's not too difficult to resolve and 0.5.2 can be released without too much hassle. Thanks for your hard work :)

**Comment by joshtayloruk** [comment]
Hey i noticed if I use EU2 and MO everything is fine. If I use TCon and MO everything is fine. But it is when I use the 3 together that there is a problem
I hope this could possibly help find the solution if you haven't already

**Comment by Muirrum** [comment]
Seems like a recurring error.

**Comment by AceologyGaming** [comment]
Just thought I'd add, I'm using it w/ ExU2 and TCon also (amoung other things)

**Comment by BrendanatorX** [comment]
Ah, I was redirected here too with the same issue, at least we know Shadow knows about it, we just gonna have to wait for a fix

**Comment by darthvader45** [comment]
Yeah, wait for a fix.  We've waited for at least a week, maybe two.  How the heck have we not gotten any progress reports on the fix?

**Comment by CJL48** [comment]
Maybe no progress reports, but there's a 0.5.2 milestone, and its now at 70%. Give him time guys.

**Comment by simeonradivoev** [comment]
Development on the mod has stopped from my side.

**Comment by darthvader45** [comment]
Well, how long has it been at 70%?  Something must be really backing this up, aside from real life.  I'll give the current dev my patience when it comes to real life, but if a set of bugs is holding them back, then they really need to ask for help.

**Comment by shadowfacts** [comment]
Real life somewhat, but mostly burn out. I worked a lot on MO for a brief period of time and exhausted myself. I'll get back to it eventually.

**Comment by darthvader45** [comment]
Okay, keep us posted on any changes to the milestone.  I'd like to be teased w/ new stuff, if anything new will be added to 0.5.2.

**Comment by Muirrum** [comment]
I'll take a look tomorrow.

**Comment by DarkEvoli** [comment]
When you can count with the new version since I am just a Modpack together.

**Comment by pbnfs** [comment]
Any updates on this?

**Comment by Muirrum** [comment]
New Version is live on the [website](http://devosmium.ddns.net/matteroverdrive/downloads.html)

**Comment by Muirrum** [comment]
https://twitter.com/DevOsmium/status/948156101777920000
